packages/did-ipfs-store/src/didStore.ts

```diff
@@ -27,9 +33,22 @@ export class DidStore implements IDidStore {
     private headers: Record<string, string> = {}
   ) {}
 
-  async save(claim: string): Promise<string> {
+  /**
+   * @param claim stringified claim. Supported types of claim content are `string` and `object`
+   * @param pinTimeout Defines how to long to wait for pinning completion before throwing error, seconds
+   */
+  async save(claim: string, pinTimeout = 5): Promise<string> {
     const blob = new Blob([claim]);
     const { cid } = await this.add(blob);
+
+    const { default: waitFor } = await (eval('import(`p-wait-for`)') as Promise<
```

Comment:
Why is `p-wait-for` necessary to be imported like this?

Reply:
It is problem we've started dealing with for some time. A lot of npm packages are migrated to ESM-only. And the only way to import ESM from CommonJs (which did-ipfs-store is) is dynamic import https://www.typescriptlang.org/docs/handbook/esm-node.html. We also wrapping `import` in `eval` to prevent typescript from changing `import` on `require`

Reply:
oh, so this is it! thx for explaining. :)